applications/vila_live/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "VILA Live",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "Python",
+		"version": "1.0.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "1.0.0",
+			"tested_versions": [
+				"1.0.0"
+			]
+		},
+		"platforms": [
+			"amd64",
+			"arm64"
+		],
+		"tags": [
+			"Large Multimodal Model",
+			"Large Vision Model"
+		],
+		"ranking": 1,
+		"dependencies": {},
```

Comment:
Please add JS dependencies here, and/or describe how `.min.js` and similar files were obtained/generated before being checked in with these changes

Reply:
Added Bootstrap dependency